### PR TITLE
Fix build

### PR DESCRIPTION
**Pull request by seadowg**
This fixes issues with the build that were caused by tests polluting each other. Key changes are:
* `ResetStateRule` now resets app depdendencies, properly clears `SharedPreferences` and deletes both the scoped and non-scoped storage directories
* `ResetStateRule` now allows you to override Dagger dependencies by passing it an `AppDependencyModule`
* `ActivityTestRule`/`IntentTestRule` should now be put in test's `RuleChain` so that `ResetStateRule` and rules definitely run before it
* Uses a new `BackgroundWorkManager` abstraction in `StorageMigrator` rather than having it talk directly to `JobManager` and `WorkManager`. This lets us test `StorageMigrator` outisde of Espresso (the Espresso tests were being flakey on CI)

#### What has been done to verify that this works as intended?

Ran locally and on test lab.

#### How does this change affect users? Describe intentional changes to behavior and behavior that could have accidentally been affected by code changes. In other words, what are the regression risks?

There are a couple of changes to `StorageMigrator` so it would be good to verify the core flows around storage migration again.

#### Before submitting this PR, please make sure you have:
- [x] run `./gradlew checkAll` and confirmed all checks still pass OR confirm CircleCI build passes and run `./gradlew connectedDebugAndroidTest` locally.
- [x] verified that any code or assets from external sources are properly credited in comments and/or in the [about file](https://github.com/opendatakit/collect/blob/master/collect_app/src/main/assets/open_source_licenses.html).
- [x] verified that any new UI elements use theme colors. [UI Components Style guidelines](https://github.com/opendatakit/collect/blob/master/CONTRIBUTING.md#ui-components-style-guidelines)

### CODE REVIEW

File: collect_app/src/androidTest/java/org/odk/collect/android/storage/StorageMigrationCompletedBannerTest.java [hunk 1]
```diff
@@ -21,18 +23,10 @@
 
 import static org.odk.collect.android.support.CollectHelpers.overrideAppDependencyModule;
 
+@RunWith(AndroidJUnit4.class)
```
Comment (lognaturel) [inline collect_app/src/androidTest/java/org/odk/collect/android/storage/StorageMigrationCompletedBannerTest.java:24]: Why is this needed? Isn't that the default runner?

**Review by lognaturel — COMMENTED** [review]
I wasn't looking at the emulator. I saw https://stackoverflow.com/a/35769764/137744 and saw that my emulator's screen was off.

<strike>When I run `StorageMigrationCompletedBannerTest` on a device that has never run the migration, all tests fail with:
```
E/SQLiteDatabase: Failed to open database '/storage/emulated/0/Android/data/org.odk.collect.android/files/metadata/instances.db'.
    android.database.sqlite.SQLiteCantOpenDatabaseException: unknown error (code 14): Could not open database
        at android.database.sqlite.SQLiteConnection.nativeOpen(Native Method)
        at android.database.sqlite.SQLiteConnection.open(SQLiteConnection.java:210)
        at android.database.sqlite.SQLiteConnection.open(SQLiteConnection.java:194)
        at android.database.sqlite.SQLiteConnectionPool.openConnectionLocked(SQLiteConnectionPool.java:493)
        at android.database.sqlite.SQLiteConnectionPool.open(SQLiteConnectionPool.java:200)
        at android.database.sqlite.SQLiteConnectionPool.open(SQLiteConnectionPool.java:192)
        at android.database.sqlite.SQLiteDatabase.openInner(SQLiteDatabase.java:864)
        at android.database.sqlite.SQLiteDatabase.open(SQLiteDatabase.java:849)
        at android.database.sqlite.SQLiteDatabase.openDatabase(SQLiteDatabase.java:750)
        at android.database.sqlite.SQLiteDatabase.openDatabase(SQLiteDatabase.java:699)
        at org.odk.collect.android.database.helpers.InstancesDatabaseHelper.databaseNeedsUpgrade(InstancesDatabaseHelper.java:228)
        at org.odk.collect.android.provider.InstanceProvider.getDbHelper(InstanceProvider.java:68)
        at org.odk.collect.android.provider.InstanceProvider.query(InstanceProvider.java:121)
        at android.content.ContentProvider.query(ContentProvider.java:1055)
        at android.content.ContentProvider.query(ContentProvider.java:1147)
        at android.content.ContentProvider$Transport.query(ContentProvider.java:240)
        at android.content.ContentResolver.query(ContentResolver.java:754)
        at android.content.ContentResolver.query(ContentResolver.java:704)
        at android.content.ContentResolver.query(ContentResolver.java:662)
        at org.odk.collect.android.dao.InstancesDao.getInstancesCursor(InstancesDao.java:242)
        at org.odk.collect.android.dao.InstancesDao.getSentInstancesCursor(InstancesDao.java:46)
        at org.odk.collect.android.activities.MainMenuActivity.countSavedForms(MainMenuActivity.java:472)
        at org.odk.collect.android.activities.MainMenuActivity.onResume(MainMenuActivity.java:307)
```

Still digging to see whether it's something about my configuration or what. The directories are correctly created but the database files never are. `SQLiteCantOpenDatabaseException` is a `SQLException` so I don't understand why it's not going in the catch block.

Oh, that SQLite exception is indeed caught and I'm just seeing it in the logs. The actual exception that's making the tests fail are 
```
androidx.test.espresso.NoActivityResumedException: No activities in stage RESUMED. Did you forget to launch the activity. (test.getActivity() or similar)?
at dalvik.system.VMStack.getThreadStackTrace(Native Method)
at java.lang.Thread.getStackTrace(Thread.java:1538)
at androidx.test.espresso.base.DefaultFailureHandler.getUserFriendlyError(DefaultFailureHandler.java:96)
at androidx.test.espresso.base.DefaultFailureHandler.handle(DefaultFailureHandler.java:59)
at androidx.test.espresso.ViewInteraction.waitForAndHandleInteractionResults(ViewInteraction.java:322)
at androidx.test.espresso.ViewInteraction.check(ViewInteraction.java:306)
at org.odk.collect.android.support.pages.MainMenuPage.assertStorageMigrationCompletedBannerIsDisplayed(MainMenuPage.java:99)
at org.odk.collect.android.storage.StorageMigrationCompletedBannerTest.when_storageMigrationCompleted_should_bannerBeVisibleAndPersistScreenRotation(StorageMigrationCompletedBannerTest.java:49)
at java.lang.reflect.Method.invoke(Native Method)
at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
at androidx.test.rule.ActivityTestRule$ActivityStatement.evaluate(ActivityTestRule.java:531)
at org.odk.collect.android.support.ResetStateRule$ResetStateStatement.evaluate(ResetStateRule.java:98)
at androidx.test.rule.GrantPermissionRule$RequestPermissionStatement.evaluate(GrantPermissionRule.java:134)
at org.junit.rules.RunRules.evaluate(RunRules.java:20)
at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
at androidx.test.ext.junit.runners.AndroidJUnit4.run(AndroidJUnit4.java:104)
at org.junit.runners.Suite.runChild(Suite.java:128)
at org.junit.runners.Suite.runChild(Suite.java:27)
at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
at org.junit.runner.JUnitCore.run(JUnitCore.java:137)
at org.junit.runner.JUnitCore.run(JUnitCore.java:115)
at androidx.test.internal.runner.TestExecutor.execute(TestExecutor.java:56)
at androidx.test.runner.AndroidJUnitRunner.onStart(AndroidJUnitRunner.java:392)
at android.app.Instrumentation$InstrumentationThread.run(Instrumentation.java:2075)
```

</strike>

**Comment by lognaturel** [comment]
Oh, that SQLite exception is indeed caught and I'm just seeing it in the logs. The actual exception that's making the tests fail are 
```
androidx.test.espresso.NoActivityResumedException: No activities in stage RESUMED. Did you forget to launch the activity. (test.getActivity() or similar)?
at dalvik.system.VMStack.getThreadStackTrace(Native Method)
at java.lang.Thread.getStackTrace(Thread.java:1538)
at androidx.test.espresso.base.DefaultFailureHandler.getUserFriendlyError(DefaultFailureHandler.java:96)
at androidx.test.espresso.base.DefaultFailureHandler.handle(DefaultFailureHandler.java:59)
at androidx.test.espresso.ViewInteraction.waitForAndHandleInteractionResults(ViewInteraction.java:322)
at androidx.test.espresso.ViewInteraction.check(ViewInteraction.java:306)
at org.odk.collect.android.support.pages.MainMenuPage.assertStorageMigrationCompletedBannerIsDisplayed(MainMenuPage.java:99)
at org.odk.collect.android.storage.StorageMigrationCompletedBannerTest.when_storageMigrationCompleted_should_bannerBeVisibleAndPersistScreenRotation(StorageMigrationCompletedBannerTest.java:49)
at java.lang.reflect.Method.invoke(Native Method)
at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
at androidx.test.rule.ActivityTestRule$ActivityStatement.evaluate(ActivityTestRule.java:531)
at org.odk.collect.android.support.ResetStateRule$ResetStateStatement.evaluate(ResetStateRule.java:98)
at androidx.test.rule.GrantPermissionRule$RequestPermissionStatement.evaluate(GrantPermissionRule.java:134)
at org.junit.rules.RunRules.evaluate(RunRules.java:20)
at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
at androidx.test.ext.junit.runners.AndroidJUnit4.run(AndroidJUnit4.java:104)
at org.junit.runners.Suite.runChild(Suite.java:128)
at org.junit.runners.Suite.runChild(Suite.java:27)
at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
at org.junit.runner.JUnitCore.run(JUnitCore.java:137)
at org.junit.runner.JUnitCore.run(JUnitCore.java:115)
at androidx.test.internal.runner.TestExecutor.execute(TestExecutor.java:56)
at androidx.test.runner.AndroidJUnitRunner.onStart(AndroidJUnitRunner.java:392)
at android.app.Instrumentation$InstrumentationThread.run(Instrumentation.java:2075)
```

File: collect_app/src/test/java/org/odk/collect/android/storage/migration/StorageMigratorTest.java [hunk 2]
```diff
@@ -27,45 +31,44 @@
...
     @Before
     public void setup() {
-        storageMigrator = spy(new StorageMigrator(storagePathProvider, storageStateProvider, storageEraser, storageMigrationRepository, generalSharedPreferences, referenceManager));
+        whenFormDownloaderIsNotRunning();
+        whenFormUploaderIsNotRunning();
 
-        doNothing().when(storageMigrator).reopenDatabases();
         doNothing().when(storageEraser).clearOdkDirOnScopedStorage();
         doNothing().when(storageEraser).deleteOdkDirFromUnscopedStorage();
         doReturn("/sdcard/odk/layers/countries/countries-raster.mbtiles").when(generalSharedPreferences).get(KEY_REFERENCE_LAYER);
```
Comment (lognaturel) [inline collect_app/src/test/java/org/odk/collect/android/storage/migration/StorageMigratorTest.java:43]: Not important to change now but there are lots of `doReturn` calls in this test file that should use the newer and more readable `when/thenReturn` syntax. See [the javadoc](https://www.javadoc.io/doc/org.mockito/mockito-core/2.7.11/org/mockito/Mockito.html#doReturn(java.lang.Object)): `Use doReturn() in those rare occasions when you cannot use when(Object).` CC @grzesiek2010

File: collect_app/src/test/java/org/odk/collect/android/storage/migration/StorageMigratorTest.java [hunk 3]
```diff
@@ -27,45 +31,44 @@
...
+    private final BackgroundWorkManager backgroundWorkManager = mock(BackgroundWorkManager.class);
 
     @Before
     public void setup() {
-        storageMigrator = spy(new StorageMigrator(storagePathProvider, storageStateProvider, storageEraser, storageMigrationRepository, generalSharedPreferences, referenceManager));
+        whenFormDownloaderIsNotRunning();
+        whenFormUploaderIsNotRunning();
 
-        doNothing().when(storageMigrator).reopenDatabases();
         doNothing().when(storageEraser).clearOdkDirOnScopedStorage();
```
Comment (lognaturel) [inline collect_app/src/test/java/org/odk/collect/android/storage/migration/StorageMigratorTest.java:41]: These `doNothing` calls shouldn't be needed because `storageEraser` is a mock. It's needed below for the `reopenDatabases` because `storageMigrator` is a `spy` and not all of its methods can be run. CC @grzesiek2010

File: collect_app/src/androidTest/java/org/odk/collect/android/support/ResetStateRule.java [hunk 4]
```diff
@@ -29,14 +56,44 @@ public Statement apply(final Statement base, Description description) {
...
+            // Setup storage location for tests
+            if (useScopedStorage) {
+                new StorageStateProvider().enableUsingScopedStorage();
+            } else {
+                new StorageStateProvider().disableUsingScopedStorage();
+            }
+
+            // Any dependencies (PropertyManager for instance) will already have been
+            // passed to JavaRosa so make sure everything is reset
+            ((Collect) context.getApplicationContext()).initializeJavaRosa();
```
Comment (lognaturel) [inline collect_app/src/androidTest/java/org/odk/collect/android/support/ResetStateRule.java:96]: I didn't realize that applications were never recreated in Espresso tests before @seadowg explained this one to me. So `Collect.onCreate` does not run for every test.

File: collect_app/src/main/java/org/odk/collect/android/activities/MainMenuActivity.java [hunk 5]
```diff
@@ -296,12 +296,8 @@ public void onClick(View v) {
                 ToastUtils.showLongToast(R.string.corrupt_settings_file_notification);
             }
         }
-    }
 
-    private void initToolbar() {
-        Toolbar toolbar = findViewById(R.id.toolbar);
-        setTitle(getString(R.string.main_menu));
-        setSupportActionBar(toolbar);
+        setUpStorageMigrationBanner();
```
Comment (lognaturel) [inline collect_app/src/main/java/org/odk/collect/android/activities/MainMenuActivity.java]: I don't think this is needed. Because of the `KEYS_WE_SHOULD_NOT_RESET` map, the scoped storage setting can't be changed by a settings file.

**Review by lognaturel — APPROVED** [review]
While I was here I took a look at some of the tests that I hadn't reviewed previously and left a couple of comments on them. I think that removing the Espresso tests for verifying the interaction between the storage migration and the background tasks is a great move. It will speed up the tests and the Robolectric tests provide as much confidence.

There are a couple of things that I think are unnecessary that I marked inline but neither is a big deal.

The changes to `StorageMigrator` are straightforward enough to review that I'd merge without QA but I'll defer that final decision to @grzesiek2010 and @opendatakit/testers. This should naturally be the next thing merged (sorry about merging things before 😭).

File: collect_app/src/main/java/org/odk/collect/android/activities/MainMenuActivity.java (hunk 5 above)
Comment (grzesiek2010) [reply collect_app/src/main/java/org/odk/collect/android/activities/MainMenuActivity.java]: It's called in onResume() and that's enough.

File: collect_app/src/androidTest/java/org/odk/collect/android/test/FormLoadingUtils.java [hunk 6]
```diff
@@ -52,6 +52,7 @@ private FormLoadingUtils() {
      */
     public static void copyFormToStorage(String formFilename, List<String> mediaFilePaths, boolean copyToDatabase) throws IOException {
         new StorageInitializer().createOdkDirsOnStorage();
+        ReferenceManager.instance().reset();
```
Comment (grzesiek2010) [inline collect_app/src/androidTest/java/org/odk/collect/android/test/FormLoadingUtils.java:55]: Shouldn't it be in ResetStateRule?

Comment (seadowg) [reply collect_app/src/androidTest/java/org/odk/collect/android/test/FormLoadingUtils.java:55]: It was originally. I actually ended up on a call with @lognaturel to double check all the JavaRosa stuff (as I'm not as knowledgable about all that) and we ended up deciding to put this here. The reasoning was that the app doesn't need `reset()` to be called (it already handles setting up the `ReferenceManager`) but `CopyFormRule` does need it.

Comment (grzesiek2010) [reply collect_app/src/androidTest/java/org/odk/collect/android/test/FormLoadingUtils.java:55]: ok

File: collect_app/src/main/java/org/odk/collect/android/utilities/ResetUtility.java [hunk 7]
```diff
@@ -35,7 +35,11 @@
 public class ResetUtility {
 
     private List<Integer> failedResetActions;
-    private final StoragePathProvider storagePathProvider = new StoragePathProvider();
+    private final StoragePathProvider storagePathProvider;
+
+    public ResetUtility() {
+        storagePathProvider = new StoragePathProvider();
```
Comment (grzesiek2010) [inline collect_app/src/main/java/org/odk/collect/android/utilities/ResetUtility.java]: What is the advantage of this change?

File: collect_app/src/test/java/org/odk/collect/android/storage/migration/StorageMigratorTest.java (hunk 3 above)
Comment (grzesiek2010) [reply collect_app/src/test/java/org/odk/collect/android/storage/migration/StorageMigratorTest.java:41]: Right probably I used spies then refactored to mocks and forgot to update here.

File: collect_app/src/test/java/org/odk/collect/android/storage/migration/StorageMigratorTest.java (hunk 2 above)
Comment (grzesiek2010) [reply collect_app/src/test/java/org/odk/collect/android/storage/migration/StorageMigratorTest.java:43]: Right probably I used spies then refactored to mocks and forgot to update here.

File: collect_app/src/main/java/org/odk/collect/android/utilities/ResetUtility.java (hunk 7 above)
Comment (seadowg) [reply collect_app/src/main/java/org/odk/collect/android/utilities/ResetUtility.java]: oooo nice catch. That's a mistake. I think I'd looked at passing one in for some reason but abandoned the idea. I'll revert that.

File: collect_app/src/main/java/org/odk/collect/android/activities/MainMenuActivity.java (hunk 5 above)
Comment (seadowg) [reply collect_app/src/main/java/org/odk/collect/android/activities/MainMenuActivity.java]: Oh my bad another mistake. I was experimenting with test coverage and left this in. @grzesiek2010 is totally right it's only needed in `onResume`

File: collect_app/src/androidTest/java/org/odk/collect/android/storage/StorageMigrationCompletedBannerTest.java (hunk 1 above)
Comment (seadowg) [reply collect_app/src/androidTest/java/org/odk/collect/android/storage/StorageMigrationCompletedBannerTest.java:24]: You technically don't need it but I think it's weird not to have it. It is possible to specify other ones and the Google team loves deprecating/changing them.

**Comment by seadowg** [comment]
@grzesiek2010 addressed the comments you found. Do you think we should go with a QA run on this?

File: collect_app/src/main/java/org/odk/collect/android/backgroundwork/CollectBackgroundWorkManager.java [hunk 8]
```diff
@@ -0,0 +1,68 @@
...
+            case ServerPollingJob.TAG: {
+                return isJobManagerWorkRunning(tag);
+            }
+
+            default:
+                throw new IllegalArgumentException("CollectBackgroundWorkManager doesn't know about " + tag);
+        }
+    }
+
+    private boolean isJobManagerWorkRunning(String tag) {
```
Comment (grzesiek2010) [inline collect_app/src/main/java/org/odk/collect/android/backgroundwork/CollectBackgroundWorkManager.java:43]: I feel anxious about the test you removed that verified those two methods. Those were android tests because starting evernote job / worker didn't work with Robolectric I don't know why. It required some time but at least it was verified and we were sure it won't cause any problem during migration. Now you moved the code here and it looks fine but generally when you refactor something it's good to add tests in this case we have refactoring and removing tests.

**Comment by grzesiek2010** [comment]
> @grzesiek2010 addressed the comments you found. Do you think we should go with a QA run on this?

as I said above I feel a bit uneasy that we no longer have test to confirm the migration can't be started if form downloader/uploader is running. Mine weren't perfect but they worked at least.

File: collect_app/src/main/java/org/odk/collect/android/backgroundwork/CollectBackgroundWorkManager.java (hunk 8 above)
Comment (seadowg) [reply collect_app/src/main/java/org/odk/collect/android/backgroundwork/CollectBackgroundWorkManager.java:43]: I can completely see where you're coming from here. The motivation behind changing the tests is that locally and on CI they were failing on and off as the waits wouldn't always work. 

I looked at faking out `JobManager`/`WorkManager` in the `StorageMigrator` tests but was unable to due to `JobManager` being a `final` class. Generally I'd wrap a depedency like this as I'm not trying to test that JobManager works but that our code that interacts with it does. Granted here we're wrapping a chunk of our own logic so it's not the "thinnest" wrapper.

We could of course test the wrapper (`CollectBackgroundWorkManager`) to give us more confidence. I'm comfortable with it being at the "edge" of our system enough that it doesn't need that but that's a pretty subjective statement. If you're feeling really uncomfortable with it I can have a stap at tests for the `CollectBackgroundWorkManager`. I'm not sure how feasible it would be for `JobManager` (although at worst we could wrap it in a veru light method proxy interface for tests) but WorkManager provides faking tools as far as I've seen.

Comment (grzesiek2010) [reply collect_app/src/main/java/org/odk/collect/android/backgroundwork/CollectBackgroundWorkManager.java:43]: Ok so let's merge it as is not to block other prs but if you have time you can add test if it's difficult please just let the QA team know about it so that they can verify this case manually one more time.

File: collect_app/src/main/java/org/odk/collect/android/backgroundwork/CollectBackgroundWorkManager.java [hunk 9]
```diff
@@ -0,0 +1,68 @@
...
+import org.odk.collect.android.tasks.ServerPollingJob;
+import org.odk.collect.android.upload.AutoSendWorker;
+import org.odk.collect.utilities.BackgroundWorkManager;
+
+import java.util.List;
+import java.util.Set;
+
+import timber.log.Timber;
+
+
```
Comment (grzesiek2010) [inline collect_app/src/main/java/org/odk/collect/android/backgroundwork/CollectBackgroundWorkManager.java]: extra empty line

**Review by grzesiek2010 — APPROVED** [review]

File: collect_app/src/main/java/org/odk/collect/android/backgroundwork/CollectBackgroundWorkManager.java (hunk 8 above)
Comment (seadowg) [reply collect_app/src/main/java/org/odk/collect/android/backgroundwork/CollectBackgroundWorkManager.java:43]: Ok that sounds good. I'll have a look at that. Go ahead and merge when you're ready!
